app/sentinel/cmd/main.go

```diff
@@ -76,10 +77,19 @@ func main() {
 		}
 	}()
 
-	safe.Post(
-		ctx,
-		*workload, *secret, *namespace, *backingStore, *useKubernetes,
-		*template, *format, *encrypt, *deleteSecret, *appendSecret, *inputKeys,
-		*notBefore, *expires,
-	)
+	safe.Post(ctx, entity.SentinelCommand{
```

Comment:
Refactored the `safe.Post()` function because ~12-13 arguments in a function were making it confusing to read/use/refactor.

Right now, it’s evident that all these parameters are part of a command to be executed.